tekton/resources/images/docker-multi-arch-template.yaml

```diff
@@ -54,9 +61,13 @@ spec:
       taskSpec:
         params:
           - name: contextPath
+            type: string
```

Comment:
@Yongxuanzhang interestingly this should be gone through mutation: https://github.com/tektoncd/pipeline/blob/a0c7d31e23f472cca39690410253a0739f3189db/pkg/apis/pipeline/v1beta1/pipeline_defaults.go#L45

Reply:
Thanks Andrew for bringing this up! We will be keeping an eye into this

Reply:
If I'm not mistaken, resources will only go through mutation if they are created after the mutation logic is deployed on the cluster. Because of that our API / Validation changes we need to account for old resources that may exist on the cluster.  
We may need a new minor release to fix this?

Reply:
One way to deal with this is to call SetDefaults() in the reconciler to ensure event the old resources have the defaults set